test/testcne/vec_test.c

```diff
@@ -135,6 +135,22 @@ test3(struct test_data *t)
     return ret;
 }
 
+static void
+result_print(tst_info_t *tst, int ret)
+{
+    if (tst == NULL) {
+        tst_error("test case struct was empty");
```

Comment:
We should add a return statement here as tst_error() does not exit or return out of this function or restructure this into some type of if else like statement.

The tst->name later will cause an exception.